modules/nf-core/delly/call/tests/main.nf.test

```diff
@@ -0,0 +1,142 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
+// nf-core modules test delly/call
+nextflow_process {
+
+    name "Test Process DELLY_CALL"
+    script "../main.nf"
+    process "DELLY_CALL"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "delly"
+    tag "delly/call"
+
+    test("homo_sapiens - [bam,bai]") {
+
+        when {
+            process {
+                """
+
+                input = [
+                    [
+                        [ id:'test' ],
+                        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_recalibrated_sorted_bam'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_recalibrated_sorted_bam_bai'], checkIfExists: true),
+                        [],
+                        [],
+                        [],
+                        ],
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta_fai'], checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
```

Comment:
Since you have multiple tests, you could add names to each of your `.match()` calls

Reply:
IMO this only increases the manual work needed to write the tests (but it's your own choice, I'm not against it). The test name is used instead if you don't use match names